### DISCUSSION TITLE
open queued workflows cannot be cancelled...and all succeeding workflows don't deploy

### DISCUSSION THREAD

**Discussion by CSU-ANRIM-Lab** [community/community#138851] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please help.   I am new in github and have this problem with open queued workflows that cannot be canceled.  Also, all succeeding workflows don't deploy.  What do you think I should do?

**Comment by origranot** [comment, marked as answer]
It's a global issue I guess.
https://www.githubstatus.com/

**Reply by LiteBrite82** [reply]
Yep, outage, we are on it.

**Reply by CSU-ANRIM-Lab** [reply]
thank you for the quick response!
